packages/amplify-category-analytics/commands/analytics/update.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
```

Comment:
We standardized on `fs-extra' in the project, please use that.

Reply:
I am seeing `readJsonFile` is available on amplify context, and this is also the approach used elsewhere, so I would just use it instead I guess?